### PR TITLE
DBZ-2152 Fix test failure

### PR DESCRIPTION
**Pull request by Naros**
https://issues.redhat.com/browse/DBZ-2152

This fixes the test failure introduced by the initial changes for DBZ-2152.

### CODE REVIEW

File: debezium-core/src/test/java/io/debezium/transforms/outbox/EventRouterTest.java [hunk 1]
```diff
@@ -799,9 +799,8 @@ public void canPassBinaryMessage() {
         assertThat(eventRouted).isNotNull();
         assertThat(eventRouted.keySchema().type()).isEqualTo(Schema.Type.STRING);
         assertThat(eventRouted.key()).isEqualTo(key);
-        assertThat(eventRouted.valueSchema().type()).isEqualTo(Schema.Type.STRUCT);
-        Struct valuePassed = (Struct) eventRouted.value();
-        assertThat(valuePassed.get("payload")).isEqualTo(value);
+        assertThat(eventRouted.valueSchema().type()).isEqualTo(Schema.Type.BYTES);
```
Comment (gunnarmorling) [inline debezium-core/src/test/java/io/debezium/transforms/outbox/EventRouterTest.java:802]: Do we have anything to add to the change notes here? Changed assertions always raise an alert for me 🚨 ;) That said, why was is a struct before?

**Comment by jpechane** [comment]
@Naros Applied, thanks

File: debezium-core/src/test/java/io/debezium/transforms/outbox/EventRouterTest.java (hunk 1 above)
Comment (jpechane) [reply debezium-core/src/test/java/io/debezium/transforms/outbox/EventRouterTest.java:802]: @gunnarmorling I am sorry but I took a liberty and merged. The changes are in the test code introduced in DBZ-2152 and was obviously copy and past of the original one

**Comment by gunnarmorling** [comment]
Thanks for merging -- still hoping for an answer re my question above :)

File: debezium-core/src/test/java/io/debezium/transforms/outbox/EventRouterTest.java (hunk 1 above)
Comment (gunnarmorling) [reply debezium-core/src/test/java/io/debezium/transforms/outbox/EventRouterTest.java:802]: Ah, ok, so it was no _existing_ assert. Yeah, all cool then. Thx!

Comment (Naros) [reply debezium-core/src/test/java/io/debezium/transforms/outbox/EventRouterTest.java:802]: What I find odd here is that the original PR with this test code passed Travis, so I didn't bother to run it locally (shame on me) to verify before rebasing & merging.  I understand why the test fails on master after the merging but why not prior during the PR phase is still weird.

**Comment by gunnarmorling** [comment]
> why not prior during the PR phase is still weird.

Indeed. Was it actually green (the checkmark) or did it remain yellow?
There seem to be issues lately where Travis fails to update the PR status,
it remains yellow, although the build already has failed long time ago.

>

**Comment by Naros** [comment]
I just double checked the PR, it was in fact green.

**Comment by gunnarmorling** [comment]
Strange, strange.

**Pull request merged**